### PR TITLE
Feature/add campaign suppression list history var

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
Pull Request
**Are you a current Fivetran customer?** 

Internal
**What change(s) does this PR introduce?** 

Add a variable `iterable__using_campaign_suppression_list_history` to disable stg_iterable__campaign_suppression_list_history if source table is not being used
**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)
Adding a variable for disabling a model
**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature (https://github.com/fivetran/dbt_iterable/issues/19)
- [ ] No 

**How did you test the PR changes?** 

 

- [x] CircleCi  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Review by fivetran-catfritz — CHANGES_REQUESTED** [review]
Hi @fivetran-reneeli! Left a couple in-line comments!

File: models/stg_iterable__campaign_suppression_list_history.sql [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{{ config(enabled=var('iterable__using_campaign_suppression_list_history', True)) }}
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply models/stg_iterable__campaign_suppression_list_history.sql:1]: ahh good catch!

File: integration_tests/dbt_project.yml [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-version: '0.2.0'
+version: '0.4.1'
 
 profile: 'integration_tests'
 
-vars:
+vars: 
+  iterable_schema: iterable_source_integration_tests_1
   iterable_source:
     iterable_schema:                          iterable_source_integration_tests
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply integration_tests/dbt_project.yml]: done!

**Review by fivetran-catfritz — APPROVED** [review]

**Comment by fivetran-catfritz** [comment]
LGTM!

**Pull request merged**